app/components/listing/feedback/FeedbackModal.jsx

```diff
@@ -102,9 +100,6 @@ const FeedbackModal = ({ service, resource, closeModal }) => {
         <img src={icon('feedback-blue-header')} alt="feedback" />
         <span>Share your Feedback</span>
       </div>
-      <div className={styles.feedbackSubheader}>
```

Comment:
As of May of 2021 We didn't have a way of reaching back out so we had to remove sub-header message "Team will usually replies within a day"